pkg/acsengine/defaults_test.go

```diff
@@ -497,6 +497,108 @@ func TestStorageProfile(t *testing.T) {
 
 }
 
+// TestMasterProfileDefaults covers tests for setMasterProfileDefaults
+func TestMasterProfileDefaults(t *testing.T) {
+	mockCS := getMockBaseContainerService("1.10.3")
```

Comment:
let's add a comment above this saying "this validates default masterProfile configuration" or something